src/utils/couponUtils.js

```
@@ -1,75 +1,28 @@
-    case 'general':
+  switch (coupon.couponType) {
+    case 'RATE':
+      discountAmount = Math.floor(orderPrice * (coupon.discountValue / 100));
+      discountAmount = Math.min(discountAmount, orderPrice);
+      break;
+
+    case 'AMOUNT':
     default:
-      // 고정 금액 할인: 주문금액 내에서만 적용
-      discountAmount = Math.min(coupon.discount, orderPrice);
+      discountAmount = Math.min(coupon.discountValue, orderPrice);
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)